cli/tests/integrations/test_node.py

```diff
@@ -240,19 +240,24 @@ def test_node_ssh_slave_with_command():
 def _node_ssh_output(args):
     cli_test_ssh_key_path = os.environ['CLI_TEST_SSH_KEY_PATH']
 
+    if os.environ.get('CLI_TEST_SSH_USER') and \
+            not any("--user" in a for a in args):
+        args.extend(['--user', os.environ.get('CLI_TEST_SSH_USER')])
+
+    if os.environ.get('CLI_TEST_MASTER_PROXY') and \
+            '--master-proxy' not in args:
+        args.append('--master-proxy')
+
     cmd = ('ssh-agent /bin/bash -c "ssh-add {} 2> /dev/null && ' +
-           'dcos node ssh --option StrictHostKeyChecking=no {}"').format(
+           'dcos node ssh --option StrictHostKeyChecking=no ' +
+           '    --option ConnectTimeout=5 {}"').format(
```

Comment:
With a ccm cluster, 5 seconds is not enough when one is a few timezones away from the us-west region. I'm currently experiencing a test which fails most of the time (if not all the time), with 8 seconds it's passing most of the time.

Could we bump this to 8? That'd still be within the 10 seconds hard sleep in `ssh_output()`.

Reply:
I've just tried from your branch and my test seems to be much less flaky, as this is only the timeout to establish the ssh connection, there are still 5 other seconds after that for a potential command to run and finish. :+1: